bundles/org.eclipse.equinox.p2.publisher.eclipse/src/org/eclipse/equinox/internal/p2/publisher/eclipse/BrandingIron.java

```diff
@@ -11,16 +11,19 @@
  *  Contributors:
  * 	IBM Corporation - initial API and implementation
  * 	Code 9 - Additional function and fixes
+ *  SAP SE - support macOS bundle URL types
```

Comment:
indentation looks a bit off

Reply:
Fixed, some weird mix of spaces and tabs in there

Reply:
Thx